<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_emp</class>
 <widget class="QWidget" name="add_emp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #34495e;</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>260</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	</string>
   </property>
   <property name="text">
    <string>تاریخ تولد:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="add_emploee">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 24px;

font-size:25px;
color:#EA2027; 
background-color:#bdc3c7; 
border: 2px solid #ff4d4d; </string>
   </property>
   <property name="text">
    <string>استخدام</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:25px;	</string>
   </property>
   <property name="text">
    <string>مشخصات را وارد کنید:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>520</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	</string>
   </property>
   <property name="text">
    <string>مرز عبور:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>470</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:20px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="last_name">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>210</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:20px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:20px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>400</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	</string>
   </property>
   <property name="text">
    <string>کد پرسنلی:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>530</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:20px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="birth_day">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>270</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:20px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>150</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
		text-align: center;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	
	text-align: center;</string>
   </property>
   <property name="text">
    <string>نام:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>200</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	</string>
   </property>
   <property name="text">
    <string>نام خوانوادگی:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 24px;

	color:#EA2027; background-color:#bdc3c7; border: 2px solid #ff4d4d; </string>
   </property>
   <property name="text">
    <string>برگشت</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>460</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	</string>
   </property>
   <property name="text">
    <string>نام کاربری:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="p_cod">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>410</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:20px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; color:#ff5500;&quot;&gt;year/month/day&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="p_cod_chose">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>410</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 24px;

	color:#EA2027; background-color:#bdc3c7; border: 2px solid #ff4d4d; </string>
   </property>
   <property name="text">
    <string>انتخاب کد</string>
   </property>
  </widget>
  <widget class="QPushButton" name="p_cod_chose_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>530</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 24px;

	color:#EA2027; background-color:#bdc3c7; border: 2px solid #ff4d4d; </string>
   </property>
   <property name="text">
    <string>انتخاب پسورد</string>
   </property>
  </widget>
  <widget class="QPushButton" name="user_chose">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>470</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 24px;

	color:#EA2027; background-color:#bdc3c7; border: 2px solid #ff4d4d; </string>
   </property>
   <property name="text">
    <string>انتخاب نام کاربری</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="basic_salari">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>330</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:20px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>320</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	</string>
   </property>
   <property name="text">
    <string>حقوق:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="accept">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_war">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>530</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red; font-size:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
